### PR TITLE
Use Dovecot repo to fix cve-2019-11500 take two

### PR DESCRIPTION
**Pull request by erik-wramner**
Switch repo and fix broken configuration.

### CODE REVIEW

**Comment by hof** [comment]
Nice work! 

Maybe remove the reference to `stretch-backport` on line 135.

**Comment by erik-wramner** [comment]
@hof, I'll do that in a separate commit to speed things up. Thanks.

@fbartels, what do you think? Shall we merge this now or wait for backports? It is running on one of my servers, so far it seems to work.

**Comment by erik-wramner** [comment]
@fbartels to save time, if you like it just merge it. If you think we should wait for backports then we'll do that instead. I'm fine with either. @tomav if you read this you naturally have a vote as well!

File: Dockerfile [hunk 1]
```diff
@@ -17,13 +17,14 @@ ENV SASLAUTHD_MECH_OPTIONS=""
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Packages
-# hadolint ignore=DL3015
+# hadolint ignore=DL3015,SC2016
```
Comment (fbartels) [inline Dockerfile]: I want to later look why the next exception is needed, but this should not block

**Comment by fbartels** [comment]
Lets get this merged then.

@tomav @erik-wramner I think we should then also really tag a new release. What do you think, should this be a v6.2.0?

**Pull request merged**